front/pages/w/[wId]/vaults/[vaultId]/apps/[aId]/runs/[runId]/index.tsx

```diff
@@ -209,7 +213,7 @@ export default function AppRun({
                 app={app}
                 run={run}
                 disabled={false}
-                url={"test"}
+                url={url}
```

Comment:
You can retrieve it directly from the client, non need to jump through the server side props since clientFacingURL is stored as NEXT_PUBLIC_DUST_CLIENT_FACING_URL